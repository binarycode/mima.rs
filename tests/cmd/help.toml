args = 'help'
status.code = 0
fs.sandbox = true
