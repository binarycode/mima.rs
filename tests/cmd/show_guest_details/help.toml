args = 'help show-guest-details'
status.code = 0
fs.sandbox = true
