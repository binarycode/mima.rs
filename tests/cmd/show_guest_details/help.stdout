Show guest details

Usage: mima show-guest-details <GUEST_ID>

Arguments:
  <GUEST_ID>  Guest ID

Options:
  -h, --help  Print help
